get-started/kube-deploy.md

```diff
@@ -7,7 +7,7 @@ description: Learn how to describe and deploy a simple application on Kubernetes
 ## Prerequisites
 
 - Download and install Docker Desktop as described in [Orientation and setup](index.md).
-- Work through containerizing an application in [Part 2](part2.md).
+- Work through containerizing an application in [Part 2](02_our_app.md).
```

Comment:
oh! just left a comment on https://github.com/docker/docker.github.io/pull/12003#issuecomment-754577462

was wondering if the new content is a "drop-in" replacement for the old one in this context 🤔

Reply:
I think some of the content got split across multiple pages, so for the generic references, linking to the first page of the set made the most sense to me. This probably needs to be reviewed on a case-by-case basis.